packages/serverless/src/model/global/user.ts

```diff
@@ -6,12 +6,13 @@ import {
   ModelGeneral,
   zkDatabaseConstant,
 } from '@zkdb/storage';
-import { ClientSession, WithoutId } from 'mongodb';
+import { ClientSession, InsertOneResult, WithoutId } from 'mongodb';
 import {
   ZKDATABASE_USER_NOBODY,
   ZKDATABASE_USER_SYSTEM,
 } from '../../common/const.js';
 import { getCurrentTime, objectToLookupPattern } from '../../helper/common.js';
+import logger from '../../helper/logger.js';
 
 export class ModelUser extends ModelGeneral<WithoutId<TUserRecord>> {
```

Comment:
Should be `OptionalId`

Reply:
I agree, change all group user and database model to OptionalId instead of WithoutId